src/basic_config.ml

```diff
@@ -15,41 +15,37 @@
 
 module Map_string = Basic_map_string
 
-type target = Wasm_gc
+type target = Wasm_gc | Riscv
```

Comment:
I think you should split this PR into at least two commits?

Reply:
Then I'll close this one and reopen with another branch.